packages/devtools_app/lib/src/app.dart

```diff
@@ -525,6 +528,67 @@ class SettingsDialog extends StatelessWidget {
   }
 }
 
+class _VerboseLoggingSetting extends StatelessWidget {
+  const _VerboseLoggingSetting();
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: const EdgeInsets.only(
+        right: 6.0,
+        bottom: 6.0,
+      ),
```

Comment:
why add this padding? It doesn't look like we have padding between the other settings? also, whenever you are adding padding/margins etc. use the values defined in theme.dart. Either defaultSpacing or denseSpacing so that we are consisten